### PR TITLE
Upstream v5.70.0

### LINKED ISSUE #4602: Upgrade terraform-provider-aws to v5.70.0
Release details: https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.70.0

### PR DESCRIPTION
**Pull request by flostadler**
- **Move upstream to v5.70.0**
- **./scripts/tidy-all.sh**
- **./scripts/patch_computed_only.sh**
- **Set up computed IDs**
- **Regen schema**
- **Regen SDK**

The detected breaking change in `getStandardsControlAssociations` is expected and benign. The output `standardsControlAssociations` snuck its way into the inputs in upstream, but it wasn't usable. This PR fixed it upstream: https://github.com/hashicorp/terraform-provider-aws/commit/f2a57042e89332fb0f910c0bd4e83ced438db38b

Additionally this corrects the `ComputedID` of `aws_iam_role_policies_exclusive`.

closes #4602

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -5848,9 +5848,18 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
...
 	prov.Resources["aws_elasticache_reserved_cache_node"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
 		return attr(state, "id"), nil
 	}
+	prov.Resources["aws_bedrock_guardrail_version"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
+		return attr(state, "guardrailArn", "version"), nil
+	}
+	prov.Resources["aws_iam_group_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
+		return attr(state, "groupName"), nil
+	}
+	prov.Resources["aws_iam_user_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
```
Comment (flostadler) [inline provider/resources.go:5862]: See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_user_policies_exclusive#import

File: provider/resources.go [hunk 2]
```diff
@@ -5848,9 +5848,18 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 		return attr(state, "functionName"), nil
 	}
 	prov.Resources["aws_iam_role_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
-		return attr(state, "role_name"), nil
+		return attr(state, "roleName"), nil
 	}
 	prov.Resources["aws_elasticache_reserved_cache_node"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
 		return attr(state, "id"), nil
 	}
+	prov.Resources["aws_bedrock_guardrail_version"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
```
Comment (flostadler) [inline provider/resources.go:5856]: See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/bedrock_guardrail_version#import

File: provider/resources.go [hunk 3]
```diff
@@ -5848,9 +5848,18 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
...
+		return attr(state, "guardrailArn", "version"), nil
+	}
+	prov.Resources["aws_iam_group_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
```
Comment (flostadler) [inline provider/resources.go:5859]: See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_group_policies_exclusive#import

**Comment by flostadler** [comment]
The detected breaking change in `getStandardsControlAssociations` is expected and benign. The output `standardsControlAssociations ` snuck its way into the inputs in upstream, but it wasn't usable. This PR fixed it upstream: https://github.com/hashicorp/terraform-provider-aws/commit/f2a57042e89332fb0f910c0bd4e83ced438db38b

File: provider/resources.go [hunk 4]
```diff
@@ -5848,9 +5848,18 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 		return attr(state, "functionName"), nil
 	}
 	prov.Resources["aws_iam_role_policies_exclusive"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
-		return attr(state, "role_name"), nil
+		return attr(state, "roleName"), nil
```
[the remark this answers is not in the document]
Comment (corymhall) [reply provider/resources.go:5851]: whoops 🤦🏼‍♂️

**Review by corymhall — APPROVED** [review]

**Pull request merged**